aspnetcore/security/authentication/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Overview of ASP.NET Core Authentication
+author: mjrousos
+description: Learn about authentication in ASP.NET Core.
+ms.author: riande
+ms.custom: mvc
+ms.date: 11/22/2019
+uid: security/authentication/index
+---
+# Overview of ASP.NET Core authentication
+
+By [Mike Rousos](https://github.com/mjrousos)
+
+Authentication is the process of determining a user's identity. [Authorization](xref:security/authorization/introduction) is the process of determining whether a user has access to a resource. In ASP.NET Core, authentication is handled by the `IAuthenticationService`, which is used by authentication [middleware](xref:fundamentals/middleware/index). The authentication service uses registered authentication handlers to complete authentication-related actions. Examples of authentication-related actions include:
+
+* Authenticating a user.
+* Responding when an unauthenticated user tries to access a restricted resource.
+
+The registered authentication handlers and their configuration options, are called "schemes".
+
+Authentication schemes are specified by registering authentication services in `Startup.ConfigureServices`. This is done by:
+
+* Calling a scheme-specific extension method after a call to `services.AddAuthentication` (such as `AddJwtBearer` or `AddCookie`, for example). These extension methods use [AuthenticationBuilder.AddScheme](xref:Microsoft.AspNetCore.Authentication.AuthenticationBuilder.AddScheme*) to register schemes with appropriate settings.
+* Less commonly, by calling [AuthenticationBuilder.AddScheme](xref:Microsoft.AspNetCore.Authentication.AuthenticationBuilder.AddScheme*) directly.
+
+For example, the following code registers authentication services and handlers for cookie and JWT bearer authentication schemes:
+
+```csharp
+services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+    .AddJwtBearer(options => Configuration.Bind("JwtSettings", options))
```

Comment:
This example would be clearer if the scheme name was also passed into AddJwtBearer and AddCookie.
```suggestion
    .AddJwtBearer(JwtBearerDefaults.AuthenticationScheme, options => Configuration.Bind("JwtSettings", options))
```

Reply:
Sure, I'll update the sample to look like this:

```csharp
services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
    .AddJwtBearer(JwtBearerDefaults.AuthenticationScheme, options => Configuration.Bind("JwtSettings", options))
    .AddCookie(CookieAuthenticationDefaults.AuthenticationScheme, options => Configuration.Bind("CookieSettings", options));
```